stories/dashboard.stories.js

```diff
@@ -88,6 +99,14 @@ storiesOf( 'Dashboard', module )
 	.add( 'Search Funnel Analytics Inactive', () => {
 		global._googlesitekitLegacyData = analyticsDashboardData;
 
+		const setupRegistry = ( { dispatch } ) => {
+			dispatch( ANALYTICS_STORE ).receiveGetSettings( {} );
+			dispatch( ANALYTICS_STORE ).setAccountID( '123456789' );
+			dispatch( ANALYTICS_STORE ).setPropertyID( '123456789' );
+			dispatch( ANALYTICS_STORE ).setInternalWebPropertyID( '123456789' );
+			dispatch( ANALYTICS_STORE ).setProfileID( '123456789' );
```

Comment:
Something missing here and below is populating the Search Console `propertyID` setting as well, otherwise these URLs are not correct. You can use `dispatch( SEARCH_CONSOLE_STORE ).receiveGetSettings( { propertyID: 'https://example.com/' } )` for that.